### PR TITLE
Dbt examples

### PR DESCRIPTION
**Pull request by krisgeus**

### CODE REVIEW

File: envs/postgres-s3-external-spark/docker-compose.yml [hunk 1]
```diff
@@ -22,20 +22,20 @@ services:
 
 
   s3server:
-    build:
-      context: ${DOCKER_CONTEXT_FOLDER}/patched-localstack
```
Comment (basvdl) [inline envs/postgres-s3-external-spark/docker-compose.yml:26]: Is it still patched?

File: envs/postgres-s3-external-spark/docker-compose.yml [hunk 2]
```diff
@@ -22,20 +22,20 @@ services:
...
     environment:
       - SERVICES=s3
+      - EDGE_PORT=4563
       - AWS_ACCESS_KEY_ID
       - AWS_SECRET_ACCESS_KEY
       - DEMO_BUCKET
       - AWS_SERVER
       - AWS_PORT
       - PORT_WEB_UI
+      - DEBUG=true
```
Comment (basvdl) [inline envs/postgres-s3-external-spark/docker-compose.yml:38]: If it does not make too much noice in the docker logs, debug is ok for me.

File: envs/postgres-s3-external-spark/docker-compose.yml (hunk 1 above)
Comment (krisgeus) [reply envs/postgres-s3-external-spark/docker-compose.yml:26]: Nope. This is the removed lines part ;-)

File: envs/postgres-s3-external-spark/docker-compose.yml (hunk 2 above)
Comment (krisgeus) [reply envs/postgres-s3-external-spark/docker-compose.yml:38]: No it doesn't. It gives you just enough noise to understand what is going on ;-)

**Pull request merged**